src/ZyRabbit.Compatibility.Legacy/ZyRabbit.Compatibility.Legacy.csproj

```diff
@@ -5,12 +5,11 @@
     <AssemblyTitle>ZyRabbit.Compatibility.Legacy</AssemblyTitle>
     <Version>2.0.0-alpha</Version>
```

Comment:
This version has to be changed as well. Also do we need this project at all? Probably it will worth to drop it

Reply:
I updated it to 1.0, let's remove projects in a separate PR, after we map each of their purpose.